geosupport/function_info/work_area_layouts/output/3-extended.csv

```diff
@@ -1,5 +1,5 @@
 name,size,from,to,comment,parent,alt_names,formatter
-List of 4 LGCs for Street 1,8,451,458,On Street,,,2
+List of 4 LGCs for Street 1,8,451,458,�On� Street,,,2

```

Comment:
are the � characters necessary here?

Reply:
No those aren't necessary. They were artifacts from copying over the left and right single quote symbols from the documentation. They can be removed.